src/components/table.js

```diff
@@ -0,0 +1,160 @@
+import RCTable from 'rc-table'
+import { Component, Fragment } from 'react'
+import { div, h, option, select } from 'react-hyperscript-helpers'
+import _ from 'underscore'
+
+/*
+* props
+* =====
+* filteredDataLength:       required
+* pageNumber:               required
+* setPageNumber:            required - function(newpageNumber)
+* setItemsPerPage:          hides selector if absent - function(newItemsPerPage)
+* itemsPerPage:             10
+* itemsPerPageOptions:      [10, 25, 50, 100]
+*/
+const paginator = function({
```

Comment:
I like that you broke this out. Let's discuss strategies for how we can reduce the duplication of properties between this component and its parents.